server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -40,6 +47,231 @@ const memberMutations = {
+            throw new Forbidden('There must be at least one admin for the account.');
+          }
+        }
+      }
+
+      // Edit member
+      let member;
+      const editableAttributes = pick(args, ['role', 'description', 'since']);
+
+      if (args.isInvitation) {
+        await models.MemberInvitation.update(editableAttributes, {
+          where: {
+            MemberCollectiveId: memberAccount.id,
+            CollectiveId: account.id,
+          },
+        });
+
+        member = await models.Member.findOne({
+          where: {
+            CollectiveId: account.id,
+            MemberCollectiveId: memberAccount.id,
+          },
+        });
+      } else {
+        await models.Member.update(editableAttributes, {
+          where: {
+            MemberCollectiveId: memberAccount.id,
+            CollectiveId: account.id,
+          },
+        });
+
+        member = await models.Member.findOne({
+          where: {
+            CollectiveId: account.id,
+            MemberCollectiveId: memberAccount.id,
+          },
+        });
+      }
+
+      return member;
+    },
+  },
+  removeMember: {
+    type: GraphQLBoolean,
+    description: 'Remove a member from the Collective',
+    args: {
+      memberAccount: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account of member to remove',
+      },
+      account: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to the Collective account',
+      },
+      role: {
+        type: MemberRole,
+        description: 'Role of member',
+      },
+      isInvitation: {
+        type: GraphQLBoolean,
+      },
+    },
+    async resolve(_, args, req) {
+      if (!req.remoteUser) {
+        throw new Unauthorized('You need to be logged in to remove a member.');
+      }
+
+      let { memberAccount, account } = args;
+
+      memberAccount = await fetchAccountWithReference(memberAccount);
+      account = await fetchAccountWithReference(account);
+
+      if (!req.remoteUser.isAdminOfCollective(account)) {
+        throw new Unauthorized('Only admins can remove a member.');
+      }
+
+      if (![MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(args.role)) {
+        throw new Forbidden('You can only remove accountants, admins, or members.');
+      }
+
+      if (args.role === MemberRoles.ADMIN) {
+        const admins = await models.Member.findAll({
+          where: {
+            CollectiveId: account.id,
+            role: MemberRoles.ADMIN,
+          },
+        });
+
+        if (admins.length === 1) {
+          if (admins[0].MemberCollectiveId === memberAccount.id) {
+            throw new Forbidden('There must be at least one admin for the account.');
+          }
+        }
+      }
+
+      // Remove member
+      if (args.isInvitation) {
+        await models.MemberInvitation.update(
+          { deletedAt: new Date() },
```

Comment:
:ice_cream: (Not a requested change): that work, but calling `.destroy` is simpler and arguably clearer